doc/spec.md

```diff
@@ -1151,35 +1151,19 @@ The only possible values for the Void type are `null` and `undefined`. The Void
 
 ### <a name="3.2.6"/>3.2.6 The Null Type
 
-The Null type corresponds to the similarly named JavaScript primitive type and is the type of the `null` literal.
+The Null type corresponds to the similarly named JavaScript primitive value and is the type of the `null` literal.
```

Comment:
Undo this. https://tc39.github.io/ecma262/#sec-terms-and-definitions-null-type

Reply:
Done